pstoremgr/pstoremgr.go

```diff
@@ -292,19 +293,28 @@ func (pm *Manager) SavePeerstoreForPeers(peers []peer.ID) error {
 // Bootstrap attempts to get up to "count" connected peers by trying those
 // in the peerstore in priority order. It returns the list of peers it managed
 // to connect to.
-func (pm *Manager) Bootstrap(count int) []peer.ID {
+func (pm *Manager) Bootstrap(count int, byPriority bool) []peer.ID {
+	totalConns := len(pm.host.Network().Conns())
+	// short-cut if there will be nothing to do.
+	if totalConns >= count {
+		return nil
+	}
```

Comment:
This is unnecessary but saves doing the sorting and reading through the peerstore.